app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/ItemListActivity.java

```diff
@@ -499,7 +500,12 @@ private void play(List<BaseItemDto> items, int ndx, boolean shuffle) {
         PlaybackLauncher playbackLauncher = KoinJavaComponent.<PlaybackLauncher>get(PlaybackLauncher.class);
         if (playbackLauncher.interceptPlayRequest(this, items.size() > 0 ? items.get(0) : null)) return;
 
+        Timber.tag(ItemListActivity.class.getSimpleName()).d("play items: %d, shuffle: %b", items.size(), shuffle);
```

Comment:
```suggestion
        Timber.d("play items: %d, shuffle: %b", items.size(), shuffle);
```

We never use the tag method

Reply:
Updated and squashed